packages/canary-react/src/stories/ic-data-table-title-bar.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs";
+
+import { IcDataTableTitleBar } from "../components";
+import { IcLink, IcButton, IcTypography, IcChip, SlottedSVG } from "@ukic/react";
+import readme from "../../../canary-web-components/src/components/ic-data-table-title-bar/readme.md";
+
+import { mdiImage } from "@mdi/js";
+
+<Meta
+  title="React Components/Data Table Title Bar"
+  component={IcDataTableTitleBar}
+  parameters={{
+    componentAPI: {
+      data: readme,
+    },
+  }}
+/>
+
+# [Canary]: Data Table Title Bar
+
+Use the Data Table Title Bar component to provide titling, description, metadata and also global table functionality.
+
+### Basic
+
+A basic example with a heading.
+
+By default, the density selector appears within the Data Table Title Bar. The three density options are:
+
+- Default
+- Dense
+- Spacious
+
+<Canvas>
+  <Story name="Basic">
+    <IcDataTableTitleBar heading="Title Bar" />
+  </Story>
+</Canvas>
+
+#### Basic code example
+
+```jsx
+import * as React from "react";
+import { IcDataTableTitleBar } from "@ukic/canary-react";
+
+const DataTableTitleBar = () => <IcDataTableTitleBar heading="Title Bar" />;
+
+export default DataTableTitleBar;
+```
+
+It is also possible to pass custom elements into the area which contains the title bar heading and description. This flexibility allows the heading tag to be updated if the heading prop is not semantically corrected based on your content structure.
```

Comment:
"if the heading prop is not semantically corrected" reads a little weird. "if the default heading level (INSERT WHAT THAT IS HERE) is not semantically correct"

Reply:
Title bar does not provide a heading tag by default so set it as
`Custom elements can be passed into the areas which contain the title bar heading and description. This flexibility allows the heading tag to be updated if the heading prop is not semantically correct based on the content structure.`